cas-management-webapp/src/main/java/org/jasig/cas/services/web/beans/RegisteredServiceEditBean.java

```diff
@@ -136,6 +182,30 @@ public static RegisteredServiceEditBean fromRegisteredService(final RegisteredSe
 
         configureAccessStrategy(svc, bean);
 
+        configureOauthSettings(svc, bean);
+
+        bean.setTheme(svc.getTheme());
+        bean.setEvalOrder(svc.getEvaluationOrder());
+        configureLogoutTypeAndUrl(svc, bean);
+
+        final RegisteredServiceUsernameAttributeProvider provider = svc.getUsernameAttributeProvider();
+        final RegisteredServiceUsernameAttributeProviderEditBean uBean = bean.getUserAttrProvider();
+
+        configureUsernameAttributeProvider(provider, uBean);
+
+        configurePublicKey(svc, bean);
+        configureProxyPolicy(svc, bean);
+        configureAttributeReleasePolicy(svc, bean);
+        return serviceBean;
+    }
+
+    /**
+     * Configure oauth settings.
+     *
+     * @param svc the svc
+     * @param bean the bean
+     */
+    private static void configureOauthSettings(final RegisteredService svc, final ServiceData bean) {
```

Comment:
I'm always suspicious with `private static`, it feels incompatible: what are the reasons behind?